src/lib/statsv2.svelte

```diff
@@ -0,0 +1,179 @@
+<!--
+  statsv2.svelte:
+    Stats highlighting the value of fixNation's work-v2
+-->
+<script lang="ts">
+  import { base } from '$app/paths';
+  import { onMount, onDestroy } from 'svelte';
+  let sections = [
+    {
+      number: '1',
```

Comment:
To match the statistic as show on the Figma:

```suggestion
      number: '#1',
```